wcon_schema.json

```diff
@@ -29,7 +29,7 @@
                 ]}
             },
             "additionalProperties": false,
-            "required": ["this", "prev", "next"]
```

Comment:
From the spec: "There must be at least one next, and they occur in temporal order. If the current file is the last in a series, the single value of next can be null, or it can be an empty array, or the next field can simply be missing."

I was going to say you are wrong to delete this requirement, but now I'm noticing the last part of the spec saying "the next field can simply be missing".

So I think the issue is with the spec which is confusing.  I say we change "There must be at least one next, and they occur in temporal order." to read "If there is a "next", it must occur in temporal order."